tests/console/yast2_cmdline.pm

```diff
@@ -44,7 +44,7 @@ sub run_yast_cli_test {
 
 sub run {
     select_console 'root-console';
-    return if (systemctl("status wicked.service", ignore_failure => 1) != 0);
+    die "wicked is not used", "The yast2-network tests can run only against wicked." if (systemctl("status wicked.service", ignore_failure => 1) != 0);
```

Comment:
Not sure why we need 2 strings here. Is it because the way it's displayed in openQA? I would suggest to put `\n` in the end of the first string as these strings will be concatenated.

Reply:
no reason actually. My mistake and i have fixed it